redis-cluster/values.yaml

```diff
@@ -42,7 +42,7 @@ kubeVersion: ""
 nameOverride: ""
 ## @param fullnameOverride String to fully override common.names.fullname
 ##
-fullnameOverride: ""
+fullnameOverride: "vote-cache"
```

Comment:
I would suggest not to add a specific name like this to the chart.

```suggestion
fullnameOverride: ""
```

Reply:
That was already in there. I removed it now.